finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/rest/BankApplication.java

```diff
@@ -10,7 +10,7 @@
  *     IBM Corporation - Initial implementation
  *******************************************************************************/
 // end::comment[]
-package global.eBank.microservices.rest;
+package io.openliberty.guides.bulkhead.global.eBank.microservices.rest;
```

Comment:
I think the rest directory should not be under microservices. The package should be 
io.openliberty.guides.bulkhead.global.eBank.rest.